generators/server/templates/pom.xml.ejs

```diff
@@ -1036,6 +1031,28 @@
                             <artifactId>validation-api</artifactId>
                             <version>${validation-api.version}</version>
                         </dependency>
+        <%_ if (devDatabaseType === 'h2Disk') { _%>
+                        <dependency>
+                            <groupId>com.h2database</groupId>
+                            <artifactId>h2</artifactId>
+                            <version>${h2.version}</version>
+                        </dependency>
+        <%_ } _%>
+                        <dependency>
+                            <groupId>org.springframework</groupId>
+                            <artifactId>spring-core</artifactId>
+                            <version>${spring.version}</version>
+                        </dependency>
+                        <dependency>
```

Comment:
Do you really need to declare the 3 dependencies? spring-context depends on spring-beans and spring-core.

Reply:
Yes it is, otherwise I wouldn't have put them.
Double checked...